src/rust/catloop/mod.rs

```diff
@@ -101,13 +104,22 @@ impl CatloopLibOS {
     pub fn new(config: &Config, runtime: SharedDemiRuntime) -> Self {
         #[cfg(feature = "profiler")]
         timer!("catloop::new");
+        let mut rng: SmallRng = SmallRng::from_entropy();
```

Comment:
On debug mode, this should be set to a fixed number, so we have some determinism?